target/linux/ath79/generic/base-files/etc/board.d/01_leds

```diff
@@ -73,6 +73,9 @@ comfast,cf-e5)
 	ucidef_set_led_rssi "rssimedium" "RSSIMEDIUM" "$boardname:blue:rssi1" "wlan0" "33" "100"
 	ucidef_set_led_rssi "rssihigh" "RSSIHIGH" "$boardname:blue:rssi2" "wlan0" "66" "100"
 	;;
+devolo,magic-2-wifi)
+	ucidef_set_led_netdev "plcw" "dLAN" "devolo:status:dlan" "eth0.1" "rx"
```

Comment:
If this is meant for PLC, don't use it for LAN.

Reply:
eth0.1 is the VLAN being used on the plc network, therefore this eth0.1 represent standard plc traffic